index.js

```diff
@@ -109,14 +89,10 @@ module.exports = {
 
   setupFactoryGuyInclude: function (app) {
     let defaultEnabled = /test|development/.test(app.env),
-      defaultSettings = { enabled: defaultEnabled, useScenarios: false },
+      defaultSettings = { enabled: defaultEnabled },
       userSettings = app.project.config(app.env).factoryGuy || {},
       settings = Object.assign(defaultSettings, userSettings);
 
-    if (settings.useScenarios) {
-      settings.enabled = true;
-    }
```

Comment:
`useScenarios` config no longer does anything - if you want to use scenarios, import and run them!